### PR TITLE
Prepare v0.4.0 release

### PR DESCRIPTION
**Pull request by owais**

### CODE REVIEW

**Review by tigrannajaryan — COMMENTED** [review]
Looks like wrong content. It is the same as Core but must be different.

**Comment by owais** [comment]
Release notes for both have been the same. This was auto-generated from githbu releases history.

**Comment by tigrannajaryan** [comment]
@owais there is a different note for Contrib: https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/untagged-66fdc54aa202b370edce

**Comment by owais** [comment]
You're right. Bad clipboard. Fixed it. PTAL @tigrannajaryan

**Comment by owais** [comment]
@dmitryax @tigrannajaryan PTAL. This is blocking the release :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+- `stackdriver` exporter: Add support for resource mapping in config (#163)
+
+## 🧰 Bug fixes 🧰
+
+- `awsxray` exporter: Wrap bad request errors for proper handling by retry queue (#205)
+- `lightstep` exporter: Ensure Lightstep exporter doesnt crash on nil node (#250)
+- `sapm` exporter: Do not break Jaeger traces before sending downstream (#193)
+- `k8s` processor: Ensure Jaeger spans work in passthrough mode (262)
+
+## 🧩 Components 🧩
```
Comment (dmitryax) [inline CHANGELOG.md:60]: Do we need to list all component in change log?

Comment (owais) [reply CHANGELOG.md:60]: We can discuss and remove post-release or in future releases. For now I suggest we continue to use the existing format to not slow things down further :)

**Review by dmitryax — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (owais) [reply CHANGELOG.md:60]: FWIW, personally I don't see much value in listing down all component every single release either. We can just list down all the new components we have each release. Not changing it right now though.

Comment (dmitryax) [reply CHANGELOG.md:60]: sounds good

**Review by jrcamp — APPROVED** [review]

**Pull request merged**